webpack.config.js

```diff
@@ -44,9 +52,9 @@ module.exports = {
             options: {
               ident: 'postcss',
               plugins: () => [
-                dashDash(),
-                 cssVariables( { preserve: true, exportVarUsagesTo: allCssVars } ),
-                 require('autoprefixer'),
+                dashDash({ mediaQueryAliases, mediaQueryAtStart: true }),
```

Comment:
I first implemented the media query prefix at the start of the variable, but then realized it also makes sense to put it right before the CSS property name. Couldn't make up my mind, so I made it configurable.